debian/ips-reverseproxy-common.install

```diff
@@ -1,2 +0,0 @@
-nmosreverseproxy* usr/lib/python2.7/dist-packages
```

Comment:
Is this file not still required, or are the relevant files moved into place by the Python packaging?

Reply:
That file is the entire python package I think, which is handled by py2dsc.